x-pack/plugins/maps/public/classes/layers/wizards/layer_wizard_registry.ts

```diff
@@ -91,3 +91,8 @@ export async function getLayerWizards(): Promise<LayerWizardWithMeta[]> {
       return wizard1.order - wizard2.order;
     });
 }
+
+export async function getWizardById(layerWizardId: string) {
```

Comment:
getWizardById does not need to be async.

Reply:
> They are all collected in the registry array, but if you try to get them without async, you will get undefined, because at the moment when you call getWizardById, the registry array has only one registered wizard, which is not equal to the id of the selected wizard.

This is a bug caused by the fact that `registerLayerWizards` is called in `MapContainer.componentDidMount`. Instead, move `registerLayerWizards` to be called in [renderApp](https://github.com/elastic/kibana/blob/main/x-pack/plugins/maps/public/render_app.tsx#L65). This will fix the timing issue you are seeing.

> I was thinking about putting wizards in the Redux store when registering, is that a good workaround?

I don't think we need to do that for this PR